docs/getting_started.rst

```diff
@@ -0,0 +1,358 @@
+download them using the downloader cog.
+
+You can start using the downloader cog by loading it: ``[p]load downloader``
+
+You can find cogs by searching on ``cogs.red``. Find whatever you want,
+there are hundreds of cogs available!
+
+.. note:: ``cogs.red``, the website that list all of the cogs is not
+    ready for v3 yet. For now, you can refer to `this post
+    <https://cogboard.red/t/approved-repositories/210>`_.
+
+.. 26-cogs not available, let's use my repo :3
+
+Cogs comes with repositories. A repository is a container of cogs
+that you can install. Let's suppose you want to install the ``say``
+cog from the repository ``Laggrons-Dumb-Cogs``. You'll first need
+to install the repository.
+
+.. code-block:: none
+
+    [p]repo add Laggrons-Dumb-Cogs https://github.com/retke/Laggrons-Dumb-Cogs
+
+.. note:: You may need to specify a branch. If so, add its name after the link.
+
+Then you can add the cog
+
+.. code-block:: none
+
+    [p]cog install Laggrons-Dumb-Cogs say
+
+Now the cog is installed, but not loaded. You can load it using the ``[p]load``
+command we talked about before.
+
+.. _getting-started-permissions:
+
+-----------
+Permissions
+-----------
+
+Red works with different levels of permissions. Every cog defines
+the level of permission needed for a command.
+
+~~~~~~~~~
+Bot owner
+~~~~~~~~~
+
+The bot owner can access all commands on every guild. He can also use
+exclusive commands that can interact with the global settings
+or system files.
+
+*You* are the owner by default. If the owner isn't set, use ``[p]set owner``.
+
+~~~~~~~~~~~~
+Server owner
+~~~~~~~~~~~~
+
+The server owner can access all commands on his guild, except the global
+ones or those who can interact with system files (available for the
+bot owner).
+
+~~~~~~~~~~~~~
+Administrator
+~~~~~~~~~~~~~
+
+The administrator is defined by its roles. You can set multiple admin roles
+with the ``[p]addadminrole`` and ``[p]removeadminrole`` commands.
+
+For example, in the mod cog, an admin can use the ``[p]modset`` command
+which defines the cog settings.
+
+~~~~~~~~~
+Moderator
+~~~~~~~~~
+
+A moderator is a step above the average users. You can set multiple moderator
+roles with the ``[p]addmodrole`` and ``[p]removemodrole`` commands.
+
+For example, in the mod cog (again), a mod will be able to mute, kick and ban;
+but he won't be able to modify the cog settings with the ``[p]modset`` command.
+
+.. tip::
+    If you don't like the default permission settings for some commands or
+    if want to restrict a cog or a command to a channel/member, you can use
+    the permissions cog.
+
+.. _getting-started-hosting:
+
+-------
+Hosting
+-------
+
+If you are hosting Red on your personal computer, you will soon notice that
+if you close the window or if you shut down you computer, Red will be offline.
+She needs an environment to work and respond.
+
+You can try to host Red somewhere he will always be online, like on a virtual
```

Comment:
I believe Red is considered a "she", but it would be best to just not use gendered pronouns.